scripts/ci/debian/debian/files

```diff
@@ -0,0 +1,1044 @@
+/home/parallels/llvm/build/mono-llvm-3.9/usr
+/home/parallels/llvm/build/mono-llvm-3.9/usr/lib
```

Comment:
No, they're not. I'm not convinced that this file is necessary since we build on the bots with this as it is. This whole .deb configuration is a bit of a hack right now. I'm using helper tools that seem to want special-snowflake configuration. Right now we just copy this directory over. This is definitely not the *right* way to do it, but it works. Most of my attention was on getting llvm itself working on linux. I put this up here to ask @directhex what route he thinks I should take to make this configuration step more sane.